netdicom/fsm.py

```diff
@@ -27,7 +27,7 @@ def AE_1(provider):
             provider.RemoteClientSocket.settimeout(provider.ConnectTimeoutSeconds)
         provider.RemoteClientSocket.connect(
             provider.primitive.CalledPresentationAddress)
-        provider.RemoteClientSocket.settimeout(timeout_original)
+        # provider.RemoteClientSocket.settimeout(timeout_original)
```

Comment:
Might be because I want the socket to stay with the new timeout beyond connecting (it to make it permanently non blocking).